specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json

```diff
@@ -1379,44 +1524,49 @@
         },
         "targetResourceName": {
```

Comment:
We do have a field "targetResource" in the same object which has the ARM id of the given resource. This is just for ease of filtering for the user.

Reply:
Can you give a scenario where a client would want to use these fields instead of just supplying the resourceId? It feels like this duplication could cause confusion.

Reply:
Also, what is the behavior in your RP if the client supplies some non-unique resourceName (for example if there are two resources with the same name but different type)?

Reply:
A common scenario is when user wants to fetch all active alerts on all resources in a resource group.

Reply:
> Also, what is the behavior in your RP if the client supplies some non-unique resourceName (for example if there are two resources with the same name but different type)?

No response will be returned in case of conflicts. Just to clarify, we are enabling users to filter on larger scope instead of a just single resource using resourceid.